mzroll/mainwindow.cpp

```diff
@@ -2885,201 +2885,54 @@ QWidget* MainWindow::eicWidgetController() {
+	QWidgetAction *btnLast = new MainWindowWidgetAction(toolBar, this,  "btnLast");
+	QWidgetAction *btnNext = new MainWindowWidgetAction(toolBar, this,  "btnNext");
+	QWidgetAction *btnPDF = new MainWindowWidgetAction(toolBar, this,  "btnPDF");
+	QWidgetAction *btnPNG = new MainWindowWidgetAction(toolBar, this,  "btnPNG");
+	QWidgetAction *btnPrint = new MainWindowWidgetAction(toolBar, this,  "btnPrint");
+	QWidgetAction *btnAutoZoom = new MainWindowWidgetAction(toolBar, this,  "btnAutoZoom");
+	QWidgetAction *btnShowTic = new MainWindowWidgetAction(toolBar, this,  "btnShowTic");
+	QWidgetAction *btnShowBarplot = new MainWindowWidgetAction(toolBar, this,  "btnShowBarplot");
+	QWidgetAction *btnShowIsotopeplot = new MainWindowWidgetAction(toolBar, this,  "btnShowIsotopeplot");
+	QWidgetAction *btnShowBoxplot = new MainWindowWidgetAction(toolBar, this,  "btnShowBoxplot");
+	QWidgetAction *btnShowSplines = new MainWindowWidgetAction(toolBar, this,  "btnShowSplines");
+
+	toolBar->addAction(btnZoom);
+	toolBar->addAction(btnBookmark);
+	toolBar->addAction(btnCopyCSV);
+	toolBar->addAction(btnMarkGood);
+	toolBar->addAction(btnMarkBad);
+	toolBar->addAction(btnGallary);
 
-
-	//TODO: Sahil-Kiran, Added while merging mainwindow
-    QToolButton *btnShowBarplot = new QToolButton(toolBar);
-    btnShowBarplot->setIcon(QIcon(rsrcPath + "/barplot.png"));
-    btnShowBarplot->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-    btnShowBarplot->setToolTip(tr("Show Barplot"));
-    btnShowBarplot->setCheckable(true);
-    btnShowBarplot->setChecked(true);
-    
-	//TODO: Sahil-Kiran, Added while merging mainwindow
-    QToolButton *btnShowBoxplot = new QToolButton(toolBar);
-    btnShowBoxplot->setIcon(QIcon(rsrcPath + "/boxplot.png"));
-    btnShowBoxplot->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-    btnShowBoxplot->setToolTip(tr("Show Boxplot"));
-    btnShowBoxplot->setCheckable(true);
-    btnShowBoxplot->setChecked(false);
-    
-	//TODO: Sahil-Kiran, Added while merging mainwindow
-    QToolButton *btnShowIsotopeplot = new QToolButton(toolBar);
-    btnShowIsotopeplot->setIcon(QIcon(rsrcPath + "/isotopeplot.png"));
-    btnShowIsotopeplot->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-    btnShowIsotopeplot->setToolTip(tr("Show Isotope Plot"));
-    btnShowIsotopeplot->setCheckable(true);
-	if (isotopePlotsDockWidget->isVisible()) {
-    	btnShowIsotopeplot->setChecked(true);
-	} else {
-		btnShowIsotopeplot->setChecked(false);
-	}
-
-	//TODO: Sahil-Kiran, Added while merging mainwindow
-    QToolButton *btnShowSplines = new QToolButton(toolBar);
-    btnShowSplines->setIcon(QIcon(rsrcPath + "/splines.png"));
-    btnShowSplines->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-    btnShowSplines->setToolTip(tr("Show Splines"));
-    btnShowSplines->setCheckable(true);
-    btnShowSplines->setChecked(false);
-	btnShowSplines->setVisible(false);
```

Comment:
This button "btnShowSplines" was not in develop branch or might be the case it was not visible. But right now in this branch this button is added(or visible) in the end on toolbar of eicWidget.